project/build.scala

```diff
@@ -146,7 +146,8 @@ object build extends Build {
     publishableSettings: _*
   ) settings (
     libraryDependencies <+= (scalaVersion)("org.scala-lang" % "scala-reflect" % _ % "provided"),
-    libraryDependencies <+= (scalaVersion)("org.scala-lang" % "scala-compiler" % _ % "provided")
+    libraryDependencies <+= (scalaVersion)("org.scala-lang" % "scala-compiler" % _ % "provided"),
+    libraryDependencies += "plugin" %% "plugin" % "0.1.0-SNAPSHOT" // TODO: change/remove once Martin's work is integrated to SM.
```

Comment:
Note: this will have to be remove prior to the pull request to be accepted (Travis will also not be happy). #140 should be accepted first.